internal/gapicgen/generator/genproto.go

```diff
@@ -73,6 +73,7 @@ func NewGenprotoGenerator(c *Config) *GenprotoGenerator {
 
 var skipPrefixes = []string{
 	"google.golang.org/genproto/googleapis/ads",
+	"google.golang.org/genproto/googleapis/storage",
```

Comment:
To confirm-- this means we will stop updating genproto/googleapis/storage, not that it will be deleted right away, correct?

Also, how will automation update storage/internal/apiv2/stubs? Will it be lumped in with the current gapic regen PR?

Reply:
> this means we will stop updating genproto/googleapis/storage, not that it will be deleted right away, correct?

Correct.

> how will automation update storage/internal/apiv2/stubs? Will it be lumped in with the current gapic regen PR?

Yes, it will be lumped in with the gapic regen PR.

Reply:
sounds good!